.github/workflows/ci.yml

```diff
@@ -117,9 +129,9 @@ jobs:
         run: cargo hack test --each-feature
         working-directory: tests-build
 
-      # Build benchmarks. Run of benchmarks is done by bench.yml workflow.
-      - name: build benches
-        run: cargo build --benches
+      # Check benchmarks. Run of benchmarks is done by bench.yml workflow.
```

Comment:
@taiki-e job `test` is taking quite long to run on windows.

Shall I split this into a separate job?

Reply:
Splitting into separate jobs seems reasonable to me.

Reply:
Done!